engines/suppliers/app/serializers/suppliers/suppliers_serializer.rb

```diff
@@ -0,0 +1,8 @@
+module Suppliers
+  class SuppliersSerializer < Shared::BaseSerializer
+    schema do
+      type 'suppliers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.